packages/contracts/contracts/CdpManagerStorage.sol

```diff
@@ -745,10 +770,14 @@ contract CdpManagerStorage is EbtcBase, ReentrancyGuard, ICdpManagerData, AuthNo
     function getSyncedDebtAndCollShares(
         bytes32 _cdpId
     ) public view returns (uint256 debt, uint256 coll) {
-        (uint256 _newColl, uint256 _newDebt, , , ) = _calcSyncedAccounting(
+        /// TODO: if feels right to sync global account here,
+        // but sync'ing global accounting here breaks some unit tests, need to investigate some more
```

Comment:
Typically this is used when global sync complete, but since we can't sync all CDP at the same time, so we provide this function to get a synced view for given CDP and `systemStEthFeePerUnitIndex` is updated by global sync